tls/s2n_kem.c

```diff
@@ -36,6 +37,7 @@ const struct s2n_kem s2n_bike_1_level_1_r1 = {
 };
 
 const struct s2n_kem s2n_sike_p503_r1 = {
+        .name = "SIKEp503r1-KEM",
```

Comment:
One name ends with "KEM" and the other doesn't, I'd prefer to stay consistent unless those names are specified from somewhere else.

Reply:
Those come from https://tools.ietf.org/html/draft-campagna-tls-bike-sike-hybrid-01#section-5.1.6 I will add a comment.